### PR TITLE
Beneficiaries appearance has been corrected

### LINKED ISSUE #1066: "Beneficiaries" coming in two line.
**Summary:**

On opening the app, the word "Beneficiaries" is coming in 2 lines. Its looking very odd.

**Steps to reproduce:**

Remove the default padding in the layout, and replace it by giving margin in the imageview.

**Expected behaviour:**

The word "Beneficiaries" should come in 1 line.

**Observed behaviour:**

The word "Beneficiaries" is coming in 2 lines.

beneficiaries

**Screenshots:**

Can be created by pressing the Volume Down and Power Button at the same time on Android 4.0 and higher.

### PR DESCRIPTION
**Pull request by atm1504**
Fixes #1066 

Please Add Screenshots If there are any UI changes.
benefic

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by therajanmaurya** [comment]
Yeah make sense, Please follow commit style guideline https://github.com/openMF/mifos-mobile/wiki/Commit-Style-Guide

**Comment by miPlodder** [comment]
@atm1504 I don't think the solution to this PR is custom. Try adding generic solution. The hard-coded value of margin are working on you device, but won't work on other devices.

File: app/src/main/res/layout/fragment_home_old.xml [hunk 1]
```diff
@@ -293,20 +293,23 @@
                         android:layout_width="match_parent"
                         android:layout_height="match_parent"
                         android:gravity="center"
-                        android:padding="@dimen/default_padding"
+
```
Comment (miPlodder) [inline app/src/main/res/layout/fragment_home_old.xml:296]: Remove this space.

File: app/src/main/res/layout/fragment_home_old.xml [hunk 2]
```diff
@@ -293,20 +293,23 @@
...
                         android:clickable="true"
                         android:id="@+id/ll_beneficiaries"
                         android:foreground="?selectableItemBackground"
                         android:orientation="vertical">
                         <ImageView
                             android:layout_width="56dp"
                             android:layout_height="56dp"
+                            android:layout_marginLeft="16sp"
```
Comment (miPlodder) [inline app/src/main/res/layout/fragment_home_old.xml:304]: Change the `sp` to `dp` for both the margins

File: app/src/main/res/layout/fragment_home_old.xml [hunk 3]
```diff
@@ -293,20 +293,23 @@
...
+                            android:layout_marginRight="16sp"
                             app:srcCompat="@drawable/ic_beneficiaries_48px"/>
                         <TextView
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
                             android:text="@string/beneficiaries"
                             android:gravity="center"
+                            android:padding="0dp"
```
Comment (miPlodder) [inline app/src/main/res/layout/fragment_home_old.xml:312]: I don't think `padding` is needed, default padding of TextView is 0dp only. Check it and make changes accordingly here.

**Review by miPlodder — CHANGES_REQUESTED** [review]
@atm1504 Make these changes.

**Comment by miPlodder** [comment]
@atm1504 Also change the Commit message to `Fix#1066 : <rest-commit-message>` as suggested by @therajanmaurya .

**Comment by prashantkh19** [comment]
@atm1504 can you do the suggested changes?

**Comment by prashantkh19** [comment]
@therajanmaurya No significant changes and the changes are also not done. I guess, we can close this.

**Pull request closed without merging**